internal/sdkv2provider/schema_cloudflare_access_mutual_tls_certificate.go

```diff
@@ -32,7 +32,7 @@ func resourceCloudflareAccessMutualTLSCertificateSchema() map[string]*schema.Sch
 			Description: "The Root CA for your certificates.",
 		},
 		"associated_hostnames": {
-			Type:     schema.TypeList,
```

Comment:
this doesn't "just work". you have to update the usage references in the CRUD methods otherwise you get a type error as the iterations are accessed differently.